newIDE/app/src/EventsSheet/ParameterFields/BehaviorField.js

```diff
@@ -19,6 +19,7 @@ export default class BehaviorField extends React.Component<
   _longDescription: ?string;
   _behaviorTypeAllowed: ?string;
   _behaviorNames: Array<string> = [];
+  // $FlowFixMe
```

Comment:
What's the issue? In 99% of cases we don't want to introduce FlowFixMe, as this hide Flow errors and reduce the potential of catching bugs in the future.

Reply:
I believe this is a general issue in the entire codebase (I might be wrong though). Flow complaints about forwardRef's typing. I couldn't come up with a solution to resolve this so I looked into ScrollView and EditorNavigator's code. 
https://github.com/4ian/GDevelop/blob/a315eabdcedb5f04b7bf287272fa46e7b43f924f/newIDE/app/src/EventsFunctionsExtensionEditor/index.js#L88-L89
https://github.com/4ian/GDevelop/blob/a315eabdcedb5f04b7bf287272fa46e7b43f924f/newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrExpressionSelector/index.js#L70-L71

Reply:
Ah you're right! :'( Forgot about that. 
I'd like to take a look but not sure to have much time... I was sure I managed to get something working at some point... should be doable especially considering that we now have the latest Flow version 🤔
Otherwise we'll go ahead without this.

Can you post the Flow error messages here for reference?

Reply:
> Can you post the Flow error messages here for reference?

Sure thing :)
Also, I had a doubt, what does the type Instance really mean? I thought it meant the 'instance' of a class for class-based components. But that couldn't be the case, right? Functional components would not have that sort of 'instance'.

Reply:
> Can you post the Flow error messages here for reference?

Cannot use AbstractComponent as a type because AbstractComponent is a value. To get the type of a value use `typeof`.

This is the error. Adding `typeof SemiControlledAutoComplete` introduces these two errors:

1. Cannot call `this._field.focus` because property `focus` is missing in  AbstractComponent [1]. Flow(InferError)
2. Cannot assign `field` to `this._field` because  `Interface` [1] is incompatible with  AbstractComponent [2]. Flow(InferError)

Reply:
Let me try a few things 🧐

Reply:
@HarsimranVirk I believe I fixed it in https://github.com/4ian/GDevelop/commit/4d63fbcca068627c2f2a1b97a558e1554f90ec91

=> export the interface and use it for the typing of the ref. :)

Reply:
Yes it solved the issue :+1: